core/transport/src/main/proto/grpc/block_worker.proto

```diff
@@ -187,6 +187,21 @@ enum TaskStatus {
   PARTIAL_FAILURE = 2;
 }
 
+message LoadFileRequest {
+  repeated File files = 1;
+}
+message File{
+  optional string ufs_path = 1;
+  // file length
+  required int64 length = 2;
```

Comment:
change to `optional`? a required field may cause backward incompatibility if we would like to remove this field in the future.

Reply:
resolved